### ISSUE TITLE
Set `tsconfig` compiler target to ES2021 to prevent `SvelteNodeViewRenderer` to break

### ISSUE THREAD

**Issue by maoosi** [sibiraj-s/svelte-tiptap#44]
I've spent hours trying to get Custom Node Extensions to work before I finally stumbled upon the solution through a comment in the project's tsconfig file. I'm raising an issue about it in case someone else encounters the same problem.

To prevent `SvelteNodeViewRenderer` from breaking when using Custom Node Extensions, you need to adjust the `compilerOptions.target` in your project's tsconfig file to `ES2021` (instead of the default `esnext`).

```ts
{
  "compilerOptions": {
    "target": "ES2021" // ES2022 or later breaks SvelteNodeViewRenderer
  }
}
```

For reference, you can track the underlying issue from tiptap repo ([tiptap issue #4906](https://github.com/ueberdosis/tiptap/issues/4906)).

**Comment by sibiraj-s** [comment]
Thank you. I gave after sometime as I didn't have enough time to spend on this.

Let's keep this open, so others don't have to spend time looking for it as you mentioned.

**Comment by sibiraj-s** [comment]
Fixed in v1.1.3.
